src/main/java/school/lemon/changerequest/java/introduction/hw1/Task2.java

```diff
@@ -3,6 +3,6 @@
 
 public class Task2 {
     public static int getMeters(int centimeters) {
-        return 0;
+        return centimeters/100;
```

Comment:
It's always better to have some variable with result before using `return` operator - you'll often need such variables for debug purposes:

```java
final int result = centimeters / 100;
return result;
```